src/server/configureChangeFeeds/memberPhaseChanged.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console, camelcase */
+import processChangeFeedWithAutoReconnect from 'rethinkdb-changefeed-reconnect'
+
+import {changefeedForMemberPhaseChanged} from 'src/server/services/dataService'
+import {handleConnectionError} from './util'
+
+export default function memberPhaseChanged(memberPhaseChangedQueue) {
+  processChangeFeedWithAutoReconnect(
+    changefeedForMemberPhaseChanged, _getFeedProcessor(memberPhaseChangedQueue), handleConnectionError,
+    {changefeedName: 'phase changed'}
```

Comment:
For consistency with other workers, should include `member` so that it's clear what type of object the phase change is related to.